lib/ts/recipe/thirdpartypasswordless/functionOverrides.ts

```diff
@@ -0,0 +1,43 @@
+import { RecipeInterface } from "supertokens-web-js/recipe/thirdpartypasswordless";
+import { RecipeOnHandleEventFunction } from "../recipeModule/types";
+import { OnHandleEventContext } from "./types";
+import { getFunctionOverrides as getThirdpartyFunctionOverrides } from "../thirdparty/functionOverrides";
+import { getFunctionOverrides as getPasswordlessFunctionOverrides } from "../passwordless/functionOverrides";
+import getThirdpartyRecipeImplementation from "./recipeImplementation/thirdPartyImplementation";
+import getPasswordlessRecipeImplementation from "./recipeImplementation/passwordlessImplementation";
+
+export const getFunctionOverrides =
+    (recipeId: string, onHandleEvent: RecipeOnHandleEventFunction<OnHandleEventContext>) =>
+    (originalImp: RecipeInterface): RecipeInterface => {
+        const thirdpartyOverrides = getThirdpartyFunctionOverrides(
+            recipeId,
+            onHandleEvent
+        )(getThirdpartyRecipeImplementation(originalImp));
+        const passwordlessOverrides = getPasswordlessFunctionOverrides(onHandleEvent)(
+            getPasswordlessRecipeImplementation(originalImp)
+        );
```

Comment:
if we are sticking to this, we should add a comment explaining what is happening here.